storage/gcloud/aio/storage/storage.py

```diff
@@ -26,23 +30,12 @@ def __init__(self, project, service_file, token=None, session=None):
                                     session=self.session,
                                     scopes=[READ_WRITE_SCOPE])
 
-    async def download(self, bucket, object_name, params=None, session=None):
-        token = await self.token.get()
-        # https://cloud.google.com/storage/docs/json_api/#encoding
-        encoded_object_name = quote(object_name, safe='')
-        url = f'{STORAGE_API_ROOT}/{bucket}/o/{encoded_object_name}'
-        headers = {
-            'Authorization': f'Bearer {token}',
-        }
+    async def download(self, bucket, object_name, session=None):
+        return await self._download(bucket, object_name,
+                                    params={'alt': 'media'}, session=session)
 
-        if not self.session:
-            self.session = aiohttp.ClientSession(conn_timeout=10,
-                                                 read_timeout=10)
-        session = session or self.session
-        resp = await session.get(url, headers=headers, params=params or {},
-                                 timeout=60)
-        resp.raise_for_status()
-        return await resp.text()
+    async def download_metadata(self, bucket, object_name, session=None):
+        return await self._download(bucket, object_name, session=session)
```

Comment:
Given we are sure that `metadata` should be `json`-encoded, it probably makes sense to return `json.loads(...)` here.